homeassistant/data_entry_flow.py

```diff
@@ -829,6 +830,14 @@ def async_show_progress(
             flow_result["step_id"] = step_id
         return flow_result
 
+    @callback
+    def async_update_progress(self, progress: float) -> None:
```

Comment:
I think we need a separate PR for this with separate `test_data_entry_flow.py` tests.